pkgs/development/python-modules/color-operations/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pytestCheckHook
+, colormath
+, cython
+, oldest-supported-numpy
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "color-operations";
+  version = "0.1.3";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "vincentsarago";
+    repo = "color-operations";
+    rev = version;
+    hash = "sha256-KsrgilcNK2ufPKrhtGdf8mdlFzhsHB2jHN+WDlZqabc=";
+  };
+
+  nativeBuildInputs = [ cython setuptools ];
+
+  propagatedBuildInputs = [ oldest-supported-numpy ];
+
+  doCheck = false;
```

Comment:
Why disabling `doCheck` is needed here ?

Reply:
Tests failed on self import:
```sh
E   ModuleNotFoundError: No module named 'color_operations.colorspace'
```
I'm not sure how to fix it right now.

Reply:
Maybe add a comment. I think this is my last suggestion to this PR. We can merge after this.

Reply:
I've found solution how to fix tests:
```nix
  preCheck = ''
    python setup.py build_ext --inplace
  '';
```